### PR TITLE
Test compiler on tut/doc 303

### PR DESCRIPTION
**Pull request by krofax**
Tested and updated compiler version in hardhat guides and tuts(quickstart, custom aa).

### CODE REVIEW

File: docs/dev/tutorials/custom-aa-tutorial.md [hunk 1]
```diff
@@ -36,23 +36,19 @@ yarn add -D typescript ts-node ethers@^5.7.2 zksync-web3 hardhat @matterlabs/har
 The current version of `zksync-web3` uses `ethers v5.7.x` as a peer dependency. An update compatible with `ethers v6.x.x` will be released soon.
 :::
 
-3. Add the zkSync and OpenZeppelin libaries.
 
-```sh
-yarn add -D @matterlabs/zksync-contracts @openzeppelin/contracts @openzeppelin/contracts-upgradeable
```
Comment (kailash-manasarovar) [inline docs/dev/tutorials/custom-aa-tutorial.md:42]: Why did you remove this line @krofax ?

**Review by kailash-manasarovar — CHANGES_REQUESTED** [review]
Sorry, I checked the ticket realised why the code is added. All good apart from that line I mentioned.

File: docs/dev/tutorials/custom-aa-tutorial.md (hunk 1 above)
Comment (krofax) [reply docs/dev/tutorials/custom-aa-tutorial.md:42]: > Sorry, I checked the ticket realised why the code is added. All good apart from that line I mentioned.

What line?

Comment (kailash-manasarovar) [reply docs/dev/tutorials/custom-aa-tutorial.md:42]: the one mentioned here ``````sh
yarn add -D @matterlabs/zksync-contracts @openzeppelin/contracts @openzeppelin/contracts-upgradeable``` etc .. that's not all of it, but that's all i can see, the whole sh code block

Comment (kailash-manasarovar) [reply docs/dev/tutorials/custom-aa-tutorial.md:42]: But you removed everything and Antonio is only asking for one to be removed?

Comment (krofax) [reply docs/dev/tutorials/custom-aa-tutorial.md:42]: Updated.

Comment (kailash-manasarovar) [reply docs/dev/tutorials/custom-aa-tutorial.md:42]: can you push the change please @krofax

**Review by kailash-manasarovar — APPROVED** [review]
